packages/kbn-scout-reporting/src/reporting/playwright.ts

```diff
@@ -60,7 +65,9 @@ export class ScoutPlaywrightReporter implements Reporter {
   }
 
   private getFileOwners(filePath: string): string[] {
-    const concatenatedOwners = getCodeOwnersForFile(filePath, this.pathsWithOwners);
+    const owners: CodeOwnership = getCodeOwnersForFile(filePath, this.pathsWithOwners);
+
+    const concatenatedOwners = owners?.teams;
```

Comment:
```suggestion
    const concatenatedOwners = getCodeOwnersForFile(filePath, this.pathsWithOwners)?.teams;
```
Same here. No need for the `CodeOwnership` type import or the `owners` variable.